File: 02576.py
min_n = 100; sum = 0
for i in range(7):
    a = int(input())

    if a % 2:
        sum += a
        min_n = min(min_n, a)

if sum == 0:
    print(-1)
else:
    print(sum, min_n, sep='\n')

File: 02576.cpp
#include <iostream>
using namespace std;

int main() {
	int min_n = 100, sum = 0;
	for (int i = 0; i < 7; i++) {
		int a; cin >> a;

		if (a % 2) {
			sum += a;
			if (a < min_n)
				min_n = a;
		}
	}

	if (sum == 0)
		cout << -1 << endl;
	else {
		cout << sum << endl;
		cout << min_n << endl;
	}
}
